Scenarios/EnablingTelemetry.md

```diff
@@ -34,7 +34,7 @@ Here is a list of the telemetry signals for different workflows:
 | Event ID | Description |
 | :-- | :-- |
 | DO0090 | AL-Go workflow ran: AddExistingAppOrTestApp |
-| DO0091 | AL-Go workflow ran: CiCd |
+| DO0091 | AL-Go workflow ran: CICD |
```

Comment:
Add DO0104 to telemetryHelper and here (as suggested otherwere) - and add the other missing entries as well:-)